### PR TITLE
documentation update

### PR DESCRIPTION
**Pull request by Rachel-nas**

### CODE REVIEW

**Review by judu — COMMENTED** [review]
Looks good to me. I'm forcing the deployment on the review app to make sure

**Comment by judu** [comment]
Review app is currently deploying on https://app-1a209bab-dd05-4c35-abe3-0b19cf27a3ac.cleverapps.io/

File: content/doc/administrate/scalability.md [hunk 1]
```diff
@@ -51,8 +51,6 @@ The following scheme depicts a Scaler replication in case of a load increase:
 
 {{< image "/images/doc/scaling_horizontal_scheme.jpg" "You can manage the range of Scalers you consider in the application configurator. The range is from 1 to 40." >}}
```
Comment (judu) [inline content/doc/administrate/scalability.md:52]: While checking this with the review app, I think that "the range is 1 to 40" is too small as a caption. I'm not sure where to put it, but currently, it's the only place that has this information. I think we should display this information differently.

For starters:

```suggestion
{{< image "/images/doc/scaling_horizontal_scheme.jpg" "Horizontal scaling: you can define the min and max numbers of Scalers you need." >}}
```

File: content/doc/administrate/scalability.md [hunk 2]
```diff
@@ -51,8 +51,6 @@ The following scheme depicts a Scaler replication in case of a load increase:
...
 
-You can manage the range of Scalers you consider in the application configurator. The range is from 1 to 20.
-
 {{< image "/images/doc/select-scalab-horizontal.png" "Horizontal scaling: the amount of scaler will increase and the sclaer size will vary between XS and S" >}}
```
Comment (judu) [inline content/doc/administrate/scalability.md:54]:
```suggestion
{{< image "/images/doc/select-scalab-horizontal.png" "Horizontal scaling: the amount and size of Scalers will be between 1 XS and 15 S." >}}
```

File: content/doc/administrate/scalability.md (hunk 1 above)
Comment (judu) [reply content/doc/administrate/scalability.md:52]: I'll put the 1 to 40 elsewhere, in another commit.

**Review by judu — CHANGES_REQUESTED** [review]

**Comment by judu** [comment]
Review app updated: https://app-1a209bab-dd05-4c35-abe3-0b19cf27a3ac.cleverapps.io/doc/administrate/scalability/

**Review by juliamrch — APPROVED** [review]

**Comment by juliamrch** [comment]
Script was not finished, checks on this PR are irrelevant and changes have been performed.

**Pull request merged**